### PR TITLE
feat: Automerge workflow

### PR DESCRIPTION
**Pull request by castastrophe**
### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Building an automerge workflow

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. TBD

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd?
- [ ] Did you update the CHANGELOG.md file with a summary of this update?

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

File: .github/workflows/automerge.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: automerge
+        uses: "pascalgn/automerge-action@ccae530ae13b6af67a7a2009c266fe925844e658"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          MERGE_LABELS: "Ready to merge, !work in progress, !needs discussion, !needs more info, !needs discovery, !needs code updates, !needs code review, !needs changelog, !needs browser testing, !needs branch testing, !Needs AT updates, !merge conflicts"
+          MERGE_REMOVE_LABELS: "Ready to merge"
+          MERGE_METHOD: "squash"
+          MERGE_COMMIT_MESSAGE: "Auto merge {pullRequest.number}"
```
Comment (mwcz) [inline .github/workflows/automerge.yml]: I haven't used Actions before so I don't know what some of these properties mean.

I'm particularly curious about `MERGE_COMMIT_MESSAGE` and how it might apply to our conventional commits practice.  There are two scenarios I can imagine:

 1. Since `MERGE_METHOD` is `"squash"`, `MERGE_COMMIT_MESSAGE` becomes the message in the squashed commit
 2. `MERGE_COMMIT_MESSAGE` is the commit message for (literal) "merge commits" which only take place when a non-fast-forward merge happens

Since `UPDATE_METHOD` is set to `"rebase"`, there should never be a merge commit because master will get rebased under every PR's branch before it gets merged into master, and due to that rebase, every merge should be a fast-forward and thus there should never be a merge commit.

tl;dr: will this impact our conventional commit practice?

File: .github/workflows/automerge.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+          MERGE_DELETE_BRANCH: "true"
```
Comment (mwcz) [inline .github/workflows/automerge.yml]: Fantastic. :D
